### PR TITLE
feat: swap and add liquidity

### PR DESCRIPTION
**Pull request by dannydaniil**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @uniswap/developer-experience
```
Comment (adjkant) [inline CODEOWNERS:1]: Why this change?

File: v3-sdk/swap-and-add-liquidity/src/example/Example.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        CurrentConfig.tokens.token1.decimals
+      )
+    ),
+    CurrencyAmount.fromRawAmount(
+      CurrentConfig.tokens.token0,
+      fromReadableAmount(
+        CurrentConfig.tokens.token0Amount,
+        CurrentConfig.tokens.token0.decimals
+      )
+    ),
```
Comment (adjkant) [inline v3-sdk/swap-and-add-liquidity/src/example/Example.tsx:91]: We do these twice, extract as setup constants? Will also make more readable for the guide!

File: v3-sdk/swap-and-add-liquidity/src/libs/conversion.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export function fromReadableAmount(amount: number, decimals: number): number {
+  return amount * Math.pow(10, decimals)
+}
+export function toReadableAmount(rawAmount: number, decimals: number): string {
+  return (rawAmount / Math.pow(10, decimals)).toString()
+}
```
Comment (adjkant) [inline v3-sdk/swap-and-add-liquidity/src/libs/conversion.ts:6]: Overflow safe ones?

**Review by adjkant — COMMENTED** [review]
Pretty much good to go sans a few org questions and conversion function latest's!

File: CODEOWNERS (hunk 1 above)
Comment (dannydaniil) [reply CODEOWNERS]: Accidental

File: v3-sdk/swap-and-add-liquidity/src/libs/conversion.ts (hunk 3 above)
Comment (dannydaniil) [reply v3-sdk/swap-and-add-liquidity/src/libs/conversion.ts:6]: Not even the balance works with the safe ones 🤔 (for pretty much all of them)

**Review by adjkant — APPROVED** [review]
